src/views/footnotes_element.ts

```diff
@@ -14,15 +14,31 @@
  * limitations under the License.
  */
 
+import { schema } from '@manuscripts/transform'
+import { findParentNodeOfTypeClosestToPos } from 'prosemirror-utils'
+
+import { getChangeClasses } from '../lib/track-changes-utils'
 import { BaseNodeProps } from './base_node_view'
 import BlockView from './block_view'
-import { createNodeOrElementView } from './creators'
+import { createNodeView } from './creators'
 
 export class FootnotesElementView<
   PropsType extends BaseNodeProps
 > extends BlockView<PropsType> {
   public elementType = 'div'
 
+  public updateContents = () => {
+    const $pos = this.view.state.doc.resolve(this.getPos())
+    const inTableFooter = !!findParentNodeOfTypeClosestToPos(
+      $pos,
+      schema.nodes.table_element_footer
+    )
+    if (inTableFooter) {
```

Comment:
you can replace that with `$pos.parent.type === schema.nodes.table_element_footer`

Reply:
Done!